docs/docs/react-without-es6.md

```diff
@@ -97,12 +97,13 @@ In React components declared as ES6 classes, methods follow the same semantics a
 class SayHello extends React.Component {
   constructor(props) {
     super(props);
+    this.state = { message: 'Hello!' };
```

Comment:
Nitpick. FB style is to write it without spaces:

```js
  this.state = {message: 'Hello!'};
```

We use this style in all other examples so it's best to keep it consistent.